profiler/snippets/snippets.go

```diff
@@ -19,16 +19,26 @@ package main
 
 import (
 	"cloud.google.com/go/profiler"
+	"google.golang.org/api/option"
 )
 
 func main() {
-	// Profiler initialization, best done as early as possible.
-	if err := profiler.Start(profiler.Config{
+	cfg := profiler.Config{
 		Service:        "myservice",
 		ServiceVersion: "1.0.0",
 		// ProjectID must be set if not running on GCP.
 		// ProjectID: "my-project",
-	}); err != nil {
+	}
+
+	// For OpenCensus users that use Cloud Trace and Cloud Monitoring:
```

Comment:
We can trim the comment - To see Profiler agent spans in Cloud Trace and Cloud Monitoring, set EnableOCTelemetry to true.

Reply:
Done. Because I noticed that not only Cloud Trace but also other APMs, such as Jaeger and Zipkin will be affected by the change, I changed the wording a bit.